README.md

```diff
@@ -1,33 +1,80 @@
+ - We base our decisions on analytics and experiments to reach consensus having heard everyone’s voice. 
+ - We measure our success through data analysis and not through the highest paid person's opinion or the loudest voice.
 
-We believe we are empowered to meet the needs of people we encounter across the Guardian.
+**Empowerment**
 
->This behaviour is consistently encouraged across all levels of engineering.
+We believe that all of us want to - and can - make a difference to the Guardian.
 
-We believe that embracing technical innovation and excellence can help us make a difference
+-Example Behaviours_: 
+  - Meeting the needs of people we encounter across the Guardian is consistently encouraged across all levels of engineering.
+  - We trust engineers to build the right things, collaborating closely and transparently with other teams.
+  - We own the problem, not the spec. We don’t wait for decisions to be made for us, we are an empowered engineering function and you are an empowered engineer. We use that power responsibly as we are conscious some decisions may need to be signed off and work collaboratively with our stakeholders.
 
-> We can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting emerging tools and solutions before others. We maintain our technical advantage through continuous innovation.
+**Technical Excellence** 
 
-We care.
+We believe that embracing technical innovation and excellence can help us make a difference.
 
->We are not driven to simply deliver the implementation of a specification, but delight in working collaboratively to achieve an objective. In return, we are enthused to do whatever we can to deliver value to the Guardian.
+_Behaviours_:
+- We think we can better achieve the Guardian's objectives by having a deep understanding of the technology we use and adopting some emerging tools and solutions before others:
+    - We have been one of the [first company in UK to adopt `scala`](https://www.theguardian.com/info/developer-blog/2013/apr/26/how-the-guardian-built-scalable-foundation)
+    - We have been one of the first company [to use continuous delivery](https://www.theguardian.com/info/developer-blog/2015/jan/05/delivering-continuous-delivery-continuously)
+    - We have been one of the [first legacy company to move entirely to the cloud (AWS)](http://perspectives.mvdirona.com/2016/03/a-decade-of-innovation/)
+    - We have been one of the pioneer of [the concept of microservices](https://martinfowler.com/articles/microservices.html#footnote-etymology)
+ - We maintain our technical advantage through continuous innovation
+ - We aim to be at the forefront of tech, staying curious, looking to innovate and continuously improve.
 
+**Knowledge sharing**
 
+We believe that it’s our responsibility to upskill one another, sharing our knowledge, learnings and mistakes. 
 
+_Behaviours_:
+ - Individuals are entitled to pairing time for their development. 
```

Comment:
It would be nice to emphasise that we also are dedicated to developing best practice for pairing. Good quality pairing that isn't passive or unequal in "burden" is really important, especially at a junior level

Reply:
I'm not sure how best to word this, but could we say that we encourage pairing, rather than that we are entitled to it?

Reply:
@buck06191 could you suggest a sentence to describe your thinking?
@jennygrahamjones Would `Individuals are encouraged to pair, as part of their development` be fine, or do you have a better suggestion?

Reply:
@jennygrahamjones thanks, I have now updated.

Reply:
Something along the lines of 
```
 - Individuals are encouraged to pair as part of their development and are empowered to take an active role in what this looks like and to find their own best practice.
```

If someone has more concrete suggestions for best practice, that could be included, but I also think it's about empowering people to take charge of how they pair and what works best for them as I've found this varies depending on individuals, teams and the work being undertaken.

Reply:
@amyhughes that captures much more eloquently what I was thinking of with 'encouraged' :)

Reply:
Sorry Josh, I've just noticed your amendment. How about a second sentence which says something like:

"We value pairing as a skill which we encourage people to grow, working together to find the styles which are most effective."

@rtyley might have something to add here

Reply:
Thanks @amyhughes for the suggestions, I have now updated the wording.